assets/app/views/main.js

```diff
@@ -86,7 +86,7 @@ var AppView = Backbone.View.extend({
     var siteEditView = new SiteEditView({ model: this.sites.get(id) });
     this.pageSwitcher.set(siteEditView);
     this.listenToOnce(siteEditView, 'site:save:success', function () {
-      this.home();
+      federalist.navigate('', { trigger: true });
```

Comment:
This was causing a bug where it was not possible to go back to the site settings view after saving the form.